### PR TITLE
Typo in format string in pyne/fortranformat/_parser.py

### LINKED ISSUE #894: Question about format string in pyne/fortranformat/_parser.py
Near the end of `parse_tokens()`, if the edit descriptor isn't matched an exception is thrown with:

```
            raise InvalidFormat('Could not identify edit descriptor '
                                'in sequence $s' % str(token_set))

```
Is `$s` a legitimate format string for `%`? It looks like a typo but I've been wrong before.

I'd prefer to refactor this to 

```
            raise InvalidFormat('Could not identify edit descriptor '
                                'in sequence {0:s}'.format(str(token_set)))

```
but I want to make sure I understand the code's intent before making that change.

### PR DESCRIPTION
**Pull request by opotowsky**
The person with a keen eye who started issue #894 noticed a typo, and this PR fixes that.

### CODE REVIEW

File: pyne/fortranformat/_parser.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def _parse_tokens(tokens, reversion=False, version=None):
         elif ed_type == 'ED10':
             ed = _read_ed10(token_set)
         else:
-            raise InvalidFormat('Could not identify edit descriptor in sequence $s' % str(token_set))
+            raise InvalidFormat('Could not identify edit descriptor in sequence %s' % str(token_set))
```
Comment (gonuke) [inline pyne/fortranformat/_parser.py]: I think the OP's suggestion of switching from the `%`-formatting to `string.format()` is probably a good idea here.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for picking up this pinoli @opotowsky

File: pyne/fortranformat/_parser.py (hunk 1 above)
Comment (opotowsky) [reply pyne/fortranformat/_parser.py]: I can do that throughout the file. Should I scrape other files for that or will that take me weeks?

Comment (opotowsky) [reply pyne/fortranformat/_parser.py]: > Thanks for picking up this pinoli @opotowsky

I drink piñon coffee every day, so I'm devoted to all low-hanging pinoli!

Comment (opotowsky) [reply pyne/fortranformat/_parser.py]: (No idea how I responded to that here)

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Thanks @opotowsky - I'm pretty sure we can merge this without the rebasing to remove tests.  If you pass more tests than necessary, it should be fine :)

**Pull request merged**